modules/client-configuration/pages/registration-bootstrap.adoc

```diff
@@ -145,6 +143,9 @@ When new packages or updates are installed on the client using {productname}, an
 To review a package EULA, open the package detail page in the {webui}.
 ====
 
+
+
+// Why do we talk about Package Locks here? ke, 2019-12-18
```

Comment:
I suspect it's here because I needed a place to add them and couldn't find anywhere better. Might be better off in a troubleshooting section maybe?

Reply:
Locking a package isn't necessarily a trouble shooting action.  But that's a better place than keeping it here.

Reply:
https://github.com/uyuni-project/uyuni-docs/issues/41